src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1005,3 +1023,147 @@ def _configure_security_policy_storage_params(arg_ctx):
+        create_args_for_complex_type(
+            c, 'parameters', ManagedInstance, [
+                'administrator_login_password',
+            ])
+
+        c.argument('administrator_login_password',
+                   options_list=['--admin-password', '-p'])
+
+        c.argument('assign_identity',
+                   options_list=['--assign-identity', '-i'],
+                   help='Generate and assign an Azure Active Directory Identity for this managed instance '
+                   'for use with key management services like Azure KeyVault. If identity is already assigned - do nothing.')
+
+    ###############################################
+    #                sql managed db               #
+    ###############################################
+    with self.argument_context('sql managed-db') as c:
+        c.argument('managed_instance_name',
+                   arg_type=managed_instance_param_type,
+                   # Allow --ids command line argument. id_part=name is 1st name in uri
+                   id_part='name')
+
+        c.argument('database_name',
+                   options_list=['--name', '-n'],
+                   help='The name of the Azure SQL Managed Database.',
+                   # Allow --ids command line argument. id_part=child_name_1 is 2nd name in uri
+                   id_part='child_name_1')
+
+    with self.argument_context('sql managed-db create') as c:
+        create_args_for_complex_type(
+            c, 'parameters', ManagedDatabase, [
+                'collation',
+            ])
+
+        c.argument('collation',
+                   required=False,
+                   help='The collation of the Azure SQL Managed Database collation to use, '
+                   'e.g.: SQL_Latin1_General_CP1_CI_AS or Latin1_General_100_CS_AS_SC')
+
+    with self.argument_context('sql managed-db restore') as c:
+        create_args_for_complex_type(
+            c, 'parameters', ManagedDatabase, [
+                'target_managed_database_name',
+                'restore_point_in_time'
+            ])
+        c.argument('target_managed_database_name',
+                   options_list=['--target-db'],
+                   required=True,
+                   help='Name of the managed database that will be created as the restore destination.')
+
+        restore_point_arg_group = 'Restore Point'
+
+        c.argument('restore_point_in_time',
+                   options_list=['--time', '-t'],
+                   arg_group=restore_point_arg_group,
+                   required=True,
+                   help='The point in time of the source database that will be restored to create the'
+                   ' new database. Must be greater than or equal to the source database\'s'
+                   ' earliestRestoreDate value. Time should be in following format: "YYYY-MM-DDTHH:MM:SS"')
+
+    with self.argument_context('sql managed-db list') as c:
+        c.argument('managed_instance_name', id_part=None)
```

Comment:
What does this do?

Reply:
This supresses the --ids parameter from sql managed-db list command as per:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-the-ids-parameter

Reply:
Can you add comment explaining what this does and why? That will help the next person to read this :)